packages/styled-system/tests/interpolation.test.ts

```diff
@@ -9,7 +9,7 @@ test("should handle array interpolations", () => {
     xl: "70em",
   })
 
-  // @ts-expect-error "&" is technically disallowed
```

Comment:
For some reason VSC shows this as a type error if I remove the line, but `tsc --noEmit` reports it as an unused line if I leave it in place. 

Same thing is happening in #3978 and I have no idea why, so just added `@ts-ignore` to turn it off completely.